### DISCUSSION TITLE
Energy Integral for Arotherm VWL105 / MEH 97

### DISCUSSION THREAD

**Discussion by Bavarialex** [john30/ebusd#623] [Q&A]
Dear all,

is there a chace to get the value of the actual energy-integral (german: Energieintegral) out of 700 or hmu?
I tested with "Integral" but got "element not found.

Unsing 08.hmu and 15.700

Thanks a lot,
Alex

**Comment by Bavarialex** [comment]
Got it.
Used 8.hmu from [mwildbolz](https://github.com/mwildbolz/ebusd-configuration)

**Reply by teove76** [reply]
Hello, I struggled a little, finally these works to read and write the Enrgy Integral minimum value (ComprStartHeatFrom), circuit 08.hmu:

r,,,ComprStartHeatFrom,,,,B51A,0511341e,ign,,IGN:3,,,,value,,SIN,,°min,
w,,,ComprStartHeatFrom,,,,B51A,0611341e,value,,SIN,,°min,
